src/app.js

```diff
@@ -5,15 +5,10 @@ export default function (app) {
     return ""
   }
 
+  var emit = {event: event, hook: hook}
```

Comment:
If we can fit in one function, a few name ideas for that one function:

- emit
- fire
- stream
- broadcast